compiler/damlc/tests/daml-test-files/DamlDocHoogle.EXPECTED.hoogle

```diff
@@ -0,0 +1,57 @@
+
+module DamlDocHoogle
+
+-- | View docs
+
+data View
+
+
+View :: View
+
+
+getController :: I -> Party
+
+-- | T docs
+
+data T
+
+-- | T docs
+
+T :: Template T => Party -> T
+
+-- | TCall docs
+
+data TCall
```

Comment:
how does `renderSimpleHoogle` handle regular (non-{template/interface/choice}) data types? would it be possible to show this as a record, including its fields?

```suggestion
data TCall = TCall
  { i :: Int
  }
  
```

Reply:
Normally hoogle won't show these, say if you look up any record on https://hoogle.haskell.org/
However, changing the txt file here to include the constructor does work, but only if its all in a single line, in which case you get this on hoogle:
![image](https://github.com/digital-asset/daml/assets/123961959/ede2f77e-7bf0-4650-a846-8c26ea1f32d1)

If we extend to say a record of 4 or 5, it gets messy:
![image](https://github.com/digital-asset/daml/assets/123961959/71f8fffd-402f-4ea1-8072-0e5a6ca4e7c4)

So we _could_, but I'm not sure if we want to?

Reply:
ah your comment got in before mine :) nah, it looks terrible even with a few fields, `data TCall` is fine :)